server/server.js

```diff
@@ -1,37 +1,82 @@
+require('newrelic');
 const express = require('express');
 const app = express();
-const PORT = 3000;
+const PORT = 8000;
 const httpProxy = require('http-proxy');
+const fetch = require('node-fetch');
 const proxy = httpProxy.createProxyServer();
 
 const photos = 'http://localhost:3001';
-const itemSummary = 'http://localhost:3002';
-const relatedProducts = 'http://localhost:3003';
-const reviews = 'http://localhost:3004';
+//const itemSummary = 'http://localhost:3002';
+//const relatedProducts = 'http://localhost:3003';
+//const reviews = 'http://localhost:3004';
 
 
 app.use('/:id', express.static('./public'));
 
-app.all('/api/photos/:id', (req, res) => {
-  proxy.web(req, res, { target: photos })
-  console.log('Connected to Photos!');
-});
+// app.all('/api/photos/:id', (req, res) => {
+//   console.log('Connected to Photos!');
+//   proxy.web(req, res, { target: photos })
+// });
 
-app.all('/api/product/:pid', (req, res) => {
-  proxy.web(req, res, { target: relatedProducts })
-  console.log('Connected to Related Products!');
-});
 
-app.all('/api/itemSummary/:id', (req, res) => {
-  proxy.web(req, res, { target: itemSummary })
-  console.log('Connected to Items!');
+app.get('/api/photos/:id', (req, res) => {
+  const { id } = req.params;
+  fetch(`http://127.0.0.1:3001/api/photos/${id}`)
+    .then(fres => fres.json())
+    .then(fres =>  {
+      res.status(200).send(fres);
+    })
+    .catch((err)=>{
+      console.log(err);
+      res.status(404).end();
+    })
```

Comment:
app.all I'll need to read more about what exactly that method does, What is the reason for changing it to app.get?

-Just a question for my learning.